### PR TITLE
[SeaTunnel #949] [sql] Add sql conf template

### PR DESCRIPTION
**Pull request by leo65535**
## Purpose of this pull request

Add sql conf template.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: config/flink.sql.conf.template [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+--
+-- This config file is a demonstration of sql processing in SeaTunnel config
+--
+-- export FLINK_HOME=/opt/flink-1.13.3
```
Comment (wuchunfu) [inline config/flink.sql.conf.template:21]: If `FLINK_HOME` is specified in the `seatunnel-env.sh file` , does it need to be specified here? There should be some clarification here, like when to specify `FLINK_HOME` here

File: config/flink.sql.conf.template [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+-- bin/start-seatunnel-sql.sh --target local -Drest.port=8081 -Dtaskmanager.numberOfTaskSlots=1 -c config/application.conf
+--
```
Comment (wuchunfu) [inline config/flink.sql.conf.template:23]: It seems more reasonable to modify `config/application.conf` to `config/application.sql`

File: config/flink.sql.conf.template (hunk 1 above)
Comment (leo65535) [reply config/flink.sql.conf.template]: Make sense.

**Comment by leo65535** [comment]
cc @CalvinKirs the workflow seems hang.

image

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**